ci-operator/step-registry/cucushift/upgrade/drop-last-hop/cucushift-upgrade-drop-last-hop-commands.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+echo "The upgrade path is:"
+cat "${SHARED_DIR}/upgrade-edge"
```

Comment:
shall we check if this file exist or not?

Reply:
if the file does not exit, the step will exit as failure because of `set -o errexit`, so do need one more pre-check.

Reply:
If this step exit, the following step will not continue, right?

Reply:
Right